src/routes/events/events.controller.js

```diff
@@ -163,11 +163,42 @@ function createResponse(req, res) {
 }
 
 /**
+ * @description Sets actual start and end dates for a specific event
+ *
+ * @param {Request} req An HTTP Request
+ * @param {Response} res An HTTP Response
+ *
+ * @returns 204 No Content
+ */
+function setActualDates(req, res) {
+	var response = req.body;
+	var id = req.params.id;
+
+	// Remove fields we don't want to update from model
+	delete event._id;
```

Comment:
@gshigeto You can get rid of these `delete` statements. They're not needed here.